component-library/src/app/pages/date-picker-documentation/date-picker-documentation.component.ts

```diff
@@ -63,22 +63,14 @@ export class DatePickerDocumentationComponent implements OnInit {
     'DatePicker.UsageInFigma.FigmaDirectionsSecondItem2'
   ];
 
-  accessibilityColorContrast: string[] = [
-    'DatePicker.Accessibility.ColorText1',
-    'DatePicker.Accessibility.ColorText2',
-    'DatePicker.Accessibility.ColorText3',
-    'DatePicker.Accessibility.ColorText4',
-    'DatePicker.Accessibility.ColorText5'
-  ];
-
   accessibilityContent: string[] = [
-    'DatePicker.Accessibility.ContentTxt1',
+    'General.AccessibilityContentItem1',
```

Comment:
Why the naming convention change?

Reply:
Because these texts are the same on DatePicker, Spinner and Input doc pages.
So I created a general key and update it with previous key